bin/pocs_shell

```diff
@@ -156,11 +156,20 @@ simulate is a space-separated list of hardware to simulate.
 Hardware names: {}   (or all for all hardware)'''.format(
             ','.join(hardware.get_all_names())))
 
+    def complete_setup_pocs(self, text, line, begidx, endidx):
```

Comment:
Can we move this down to the other Utility methods section? The organization scheme is entirely my own across a lot of our files but I think makes sense. Let me know if disagree.

Reply:
I only see the `text` param being used. Are the others still required?

Reply:
They are all passed in by Cmd when it asks for a set of completions. I assume that Python would complain if we didn't declare them.

Reply:
Regarding moving the completion_* (and help_*) methods, I'd like to keep those methods grouped together by command name. Perhaps we should add a delineating comment (i.e. ####...) between commands?